Tools/autotest/common.py

```diff
@@ -2438,6 +2438,19 @@ def assert_bytes_equal(self, bytes1, bytes2, maxlen=None):
             if bytes1[i] != bytes2[i]:
                 raise NotAchievedException("differ at offset %u" % i)
 
+    def HIGH_LATENCY2(self):
+        '''test sending of HIGH_LATENCY2'''
+        # should not be getting HIGH_LATENCY2 by default
+        m = self.mav.recv_match(type='HIGH_LATENCY2', blocking=True, timeout=2)
+        if m is not None:
+            raise NotAchievedException("Shouldn't be getting HIGH_LATENCY2 by default")
+        if (m.failure_flags & mavutil.mavlink.HL_FAILURE_FLAG_GPS) != 0:
```

Comment:
m is None and we read from it ?

Reply:
Forgot to fill that in from a freshly polled message!